src/main/scala/arcana/world/Main.scala

```diff
@@ -2,7 +2,8 @@ package arcana.world
 
 object Main {
   def main(args: Array[String]): Unit = {
-    System.out.println("Hello World")
+    var version = "0.0";

```

Comment:
```suggestion
    val version = "0.0";
```
Do avoid using mutable variables.